lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/MealsAdapter.kt

```diff
@@ -13,11 +13,9 @@ import com.kcc.kmmhackathon.shared.entity.Meal
 
 class MealsAdapter (var mealsList: List<Meal>) : RecyclerView.Adapter<MealsAdapter.MealsViewHolder>() {
 
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MealsViewHolder {
-        return LayoutInflater.from(parent.context)
-            .inflate(R.layout.meal_item_row, parent, false)
-            .run(::MealsViewHolder)
```

Comment:
So, when I moved the file this wouldn't compile and I had to change it. i'd like to change it back if anyone can figure out why it stopped compiling!

Reply:
What was the error?

Reply:
Dunno but it works now lol